pkg/controller/controller_binding.go

```diff
@@ -621,28 +676,35 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 			return err
 		}
 
-		c.setServiceInstanceCredentialCondition(
-			toUpdate,
-			v1alpha1.ServiceInstanceCredentialConditionReady,
-			v1alpha1.ConditionFalse,
-			successUnboundReason,
-			"The binding was deleted successfully",
-		)
-		c.clearServiceInstanceCredentialCurrentOperation(toUpdate)
+		if toUpdate.Status.OrphanMitigationInProgress {
+			s := fmt.Sprintf(`Orphan mitigation successful for ServiceInstanceCredential "%v/%v", setting condition to ready/false`, binding.Namespace, binding.Name)
+			c.setServiceInstanceCredentialCondition(toUpdate,
+				v1alpha1.ServiceInstanceCredentialConditionReady,
+				v1alpha1.ConditionFalse,
+				successOrphanMitigationReason,
+				s)
```

Comment:
The message for the Ready/False condition does not need to say that it set the condition to ready/false.